src/js/claims-status/utils/appeals-v2-helpers.jsx

```diff
@@ -845,26 +837,25 @@ export function getNextEvents(currentStatus, details) {
       const ssocDuration = makeDurationText(details.ssocTimeliness);
       const formattedSocDate = moment(details.lastSocDate, 'YYYY-MM-DD').format('MMMM Do, YYYY');
       return  {
-        header: 'What happens next depends on whether you send in new evidence.',
+        header: 'What happens next depends on whether you submit new evidence.',
         events: [
           {
             title: 'Your appeal will be sent to the Board',
             description: (
               <p>
-                <strong>If you don’t send in new evidence after the Statement of the Case
-                on {formattedSocDate}</strong>, the Decision Review Officer will finish
+                <strong>If you don’t submit new evidence after the Supplemental Statement of the Case on {formattedSocDate}</strong>, the Decision Review Officer will finish
                 their review and transfer your case to the Board of Veterans’ Appeals.
               </p>
             ),
             durationText: certDuration.header,
             cardDescription: `The Veterans Benefits Administration typically takes ${certDuration.description} to transfer cases to the Board.`
           }, {
-            title: 'The Veterans Benefits Administration will send you a new Statement of the Case',
+            title: 'The Veterans Benefits Administration will send you a Supplemental Statement of the Case',
             description: (
               <p>
-                <strong>If you send new evidence after the Statement of the Case
+                <strong>If you send new evidence after the Supplemental Statement of the Case
                 on {formattedSocDate}</strong>, the Decision Review Officer will
-                need to write a new Statement of the Case before transferring your
+                need to write a Supplemental Statement of the Case before transferring your
```

Comment:
"...after the Supplemental Statement of the Case...the Decision Review Officer will need to write a Supplemental Statement of the Case..."

@gnakm This reads a little funny to this layman. Would it make sense to say "...write a **new** Supplemental Statement of the Case..."?

Reply:
Ah good 👀Yes it should be `write a new Supplemental Statement of the Case"